skore/src/skore/sklearn/_comparison/report.py

```diff
@@ -0,0 +1,162 @@
+from __future__ import annotations
+
+import time
+from collections.abc import Iterable
+from typing import Optional, Union
+
+import joblib
+import numpy as np
+
+from skore.externals._pandas_accessors import DirNamesMixin
+from skore.sklearn._base import _BaseReport
+from skore.sklearn._estimator.report import EstimatorReport
+
+
+class ComparisonReport(_BaseReport, DirNamesMixin):
+    """Report for comparison of instances of :class:`skore.EstimatorReport`.
+
+    Caution: reports passed to `ComparisonReport` are not copied. If you pass
+    a report to `ComparisonReport`, and then modify the report outside later, it will
+    affect the report stored inside the `ComparisonReport` as well, which can lead to
+    inconsistent results. For this reason, modifying reports after creation is strongly
+    discouraged.
+
+    Parameters
+    ----------
+    reports : list of :class:`~skore.EstimatorReport` instances or dict
+        Estimator reports to compare.
+
+        * If `reports` is a list, the class name of each estimator is used.
+        * If `reports` is a dict, it is expected to have estimator names as keys \
+        and :class:`~skore.EstimatorReport` instances as values. \
+        If the keys are not strings, they will be converted to strings.
```

Comment:
```suggestion
        * If `reports` is a dict, it is expected to have estimator names as keys
          and :class:`~skore.EstimatorReport` instances as values.
          If the keys are not strings, they will be converted to strings.
```

No need for the `\`, you only need to align the next line under the text.

Reply:
done, thanks!